pom.xml

```diff
@@ -89,6 +89,9 @@
         <revision>0.8</revision>
         <changelist>-SNAPSHOT</changelist>
         <jenkins.version>2.346.3</jenkins.version>
+        <spotbugs.effort>Max</spotbugs.effort>
+        <spotbugs.failOnError>true</spotbugs.failOnError>
```

Comment:
The failOnError setting is already the default from the parent pom.  Don't need to set it here.
```suggestion
```

Reply:
I'm not sure why you marked the conversation as resolved.  I've "unresolved" it so that I can apply the change.